Jenkinsfile

```diff
@@ -406,10 +405,33 @@ stage('Build with sync') {
   )
 }
 
+stage ('Build .NET Core with sync') {
+  nodeWithCleanup('windows') {
+    getArchive()
+    unstash 'macos-wrappers-sync'
+    unstash 'linux-wrappers-sync'
+    unstash 'tools-weaver'
+
+    bat 'dotnet restore Tests/Tests.NetCore/Tests.NetCore.csproj'
```

Comment:
Not sure why, but it is. `dotnet restore` creates a `.targets` file that imports the Fody targets, whereas I wasn't able to see such created by `msbuild /t:Restore`. Consequently, if we don't run `dotnet restore`, the weaver is not run.